CHANGES.md

```diff
@@ -14,7 +14,10 @@
 [Unreleased](https://github.com/bird-house/birdhouse-deploy/tree/master) (latest)
 ------------------------------------------------------------------------------------------------------------------
 
-[//]: # (list changes here, using '-' for each new entry, remove this when items are added)
+## Fixes
+
+- Add missing ``config/canarie-api/weaver_config.py`` entry to ``.gitignore`` of ``./components/weaver``
+  that is generated from the corresponding template file.
```

Comment:
@fmigneault add a note to this changelog that "Even with this autodeploy fix, autodeploy will not resume automatically.  A manual git pull will be required.  This fix is to fix subsequent autodeploy trigger only".